sdlib/dmd/gc.d

```diff
@@ -67,45 +67,59 @@ bool __sd_gc_fetch_alloc_info(void* ptr, void** base, size_t* size,
 	return true;
 }
 
-size_t __sd_gc_get_array_used(void* ptr, PageDescriptor pd) {
+void[] __sd_gc_get_allocation_slice(const void* ptr) {
+	return threadCache.getAllocationSlice(ptr);
+}
```

Comment:
This can go in on its own, no need to package it.

That being said, if we are going to have the size off by one for appendable, this needs to be off by one as well.

Reply:
removed